code/parse/sexp.cpp

```diff
@@ -37520,6 +37537,12 @@ SCP_vector<sexp_help_struct> Sexp_help = {
 		"\tRest:\tHUD Gauge Group name\r\n"
 	},
 
+	{OP_HUD_FORCE_SENSOR_STATIC, "hud-force-sensor-static\r\n"
+		"\tActivates or deactivates hud static as if sensors are damaged."
+		"Takes 1 Arguments...\r\n"
+		"\t1:\tBoolean, whether or not to display this gauge\r\n"
```

Comment:
"display this gauge" isn't really accurate as an argument description for this sexp.  Also "argument" should be singular.

Reply:
Done